pocs/tests/test_camera.py

```diff
@@ -297,14 +301,18 @@ def test_exposure_not_connected(camera):
     camera._connected = True
 
 
-def test_observation(camera):
+def test_observation(camera, images_dir):
     """
     Tests functionality of take_observation()
     """
     field = Field('Test Observation', '20h00m43.7135s +22d42m39.0645s')
     observation = Observation(field, exp_time=1.5 * u.second)
+    observation.seq_time = 'seq_time'
     camera.take_observation(observation, headers={})
     time.sleep(7)
+    observation_pattern = os.path.join(images_dir, 'fields', 'TestObservation',
```

Comment:
Note that you can now 

```
os.path.join(
    observation.directory, 
    camera.uid, 
    observation.seq_time,
    '*.fits*'
)
```
Unless the `images_dir` from the fixture is pointing somewhere else.

Reply:
`images_dir` here is the `/tmp/pytest-of-<username>/pytest-N` directory that comes from the temporary directory fixture. I'll have to have a think about whether `observation.directory` would be the same here.

Reply:
I'm pretty sure those left over images are coming from the tests in `test_observatory.py`, otherwise these tests (which are checking for FITS files appearing in the temporary directory) would be failing.